delivery_roulier_chronopost_fr/models/stock_picking.py

```diff
@@ -0,0 +1,35 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import date, datetime
+
+from odoo import models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _chronopost_fr_get_auth(self, account, package=None):
+        vals = self._roulier_get_auth(account, package=package)
+        if account.chronopost_fr_subaccount:
+            vals.update({"subAccount": account.chronopost_fr_subaccount})
+        return vals
+
+    def _chronopost__fr_get_shipping_date(self, package=None):
+        return date.today()
```

Comment:
Should use odoo.fields.Date as we are in an Odoo context

Reply:
@florian-dacosta Please check that and we can process.

Reply:
Hello,
I don't really get this point, since this date is meant to be sent to an external system (roulier) and not Odoo.
I've done it anyway.